languages/python/flask-oso/requirements.txt

```diff
@@ -1,2 +1,2 @@
-oso~=0.10.0
+oso==0.11.0-alpha
```

Comment:
Is this intentional?

Reply:
Yeah, tests were failing without. Didn't investigate much deeper though